shared/appengine.go

```diff
@@ -11,16 +11,53 @@ import (
 	"fmt"
 	"net/http"
 	"net/url"
-	"os/exec"
-	"strings"
+	"os"
 	"time"
 
 	"github.com/google/go-github/v31/github"
 	"golang.org/x/oauth2"
-	"google.golang.org/appengine"
+	apps "google.golang.org/api/appengine/v1"
+
+	// TODO(#1747): Deprecate this library.
 	"google.golang.org/appengine/user"
 )
 
+// runtimeIdentity contains the identity of the current AppEngine service when
+// running on GAE, or empty when running locally.
+var runtimeIdentity struct {
+	LocationID  string
```

Comment:
This appears unused in this change; is it used in a dependent change?

Reply:
Yeah, Cloud Tasks requires this in the API call.